openlibrary/plugins/upstream/models.py

```diff
@@ -97,7 +97,7 @@ def get_prev(self):
         return editions[i - 1]
 
     def get_covers(self):
-        return [Image(self._site, 'b', c) for c in self.covers if c > 0]
+        return [Image(self._site, 'b', c) for c in self.covers if c and c > 0]
```

Comment:
Let's add a comment here that it can be `null` or `-1` (both of which are errors). That way once we fix the underlying data, we can remove the extra checks.

Reply:
@cdrini And "fix the underlying data" also includes "fix the processes that created the errors" and "test that erroneous data cannot be saved", non?

Reply:
@cdrini Done!

Reply:
@LeadSongDog The process was fixed in https://github.com/internetarchive/openlibrary/pull/2838 ; I don't believe there's a way for a user to save data with covers=null, so this should be possible after #2838 .